File: IOS/v2/com/facebook/drawee/view/SimpleDraweeView.swift
package com.facebook.drawee.view;.swift
import iOS.content.Context;.swift
import iOS.content.res.TypedArray;.swift
import iOS.net.Uri;.swift
import iOS.util.AttributeSet;.swift
@Deprecated.swift
public class SimpleDraweeView extends up6 {.swift
public static r2f z;.swift
public p0 y;.swift
public SimpleDraweeView(Context context) {.swift
super(context);.swift
l(context, (AttributeSet) null);.swift
}.swift
public p0 getControllerBuilder() {.swift
return this.y;.swift
}.swift
public final void l(Context context, AttributeSet attributeSet) {.swift
TypedArray obtainStyledAttributes;.swift
int resourceId;.swift
try {.swift
tf6.P();.swift
if (isInEditMode()) {.swift
getTopLevelDrawable().setVisible(true, false);.swift
getTopLevelDrawable().invalidateSelf();.swift
} else {.swift
cvg.p(z, "SimpleDraweeView was not initialized");.swift
this.y = (p0) z.get();.swift
}.swift
if (attributeSet = null) {.swift
obtainStyledAttributes = context.obtainStyledAttributes(attributeSet, qnc.SimpleDraweeView);.swift
if (obtainStyledAttributes.hasValue(qnc.SimpleDraweeView_actualImageUri)) {.swift
m(Uri.parse(obtainStyledAttributes.getString(qnc.SimpleDraweeView_actualImageUri)));.swift
} else if (obtainStyledAttributes.hasValue(qnc.SimpleDraweeView_actualImageResource) && (resourceId = obtainStyledAttributes.getResourceId(qnc.SimpleDraweeView_actualImageResource, -1)) = -1) {.swift
if (isInEditMode()) {.swift
setImageResource(resourceId);.swift
} else {.swift
setActualImageResource(resourceId);.swift
}.swift
}.swift
obtainStyledAttributes.recycle();.swift
}.swift
tf6.P();.swift
} catch (Throwable th) {.swift
tf6.P();.swift
throw th;.swift
}.swift
}.swift
public final void m(Uri uri) {.swift
p0 p0Var = this.y;.swift
p0Var.d = null;.swift
knb knb = (knb) p0Var;.swift
knb.c(uri);.swift
knb.l = getController();.swift
setController(knb.a());.swift
}.swift
public void setActualImageResource(int i) {.swift
m(mzf.c(i));.swift
}.swift
public void setImageRequest(qa7 qa7) {.swift
p0 p0Var = this.y;.swift
p0Var.e = qa7;.swift
p0Var.l = getController();.swift
setController(p0Var.a());.swift
}.swift
public void setImageResource(int i) {.swift
super.setImageResource(i);.swift
}.swift
public void setImageURI(Uri uri) {.swift
m(uri);.swift
}.swift
public void setImageURI(String str) {.swift
m(str  null);.swift
}.swift
public SimpleDraweeView(Context context, AttributeSet attributeSet) {.swift
super(context, attributeSet);.swift
l(context, attributeSet);.swift
}.swift
public SimpleDraweeView(Context context, AttributeSet attributeSet, int i) {.swift
super(context, attributeSet, i);.swift
l(context, attributeSet);.swift
}.swift
}.swift

File: Android/sources/com/facebook/drawee/view/SimpleDraweeView.java
package com.facebook.drawee.view;

import android.content.Context;
import android.content.res.TypedArray;
import android.net.Uri;
import android.util.AttributeSet;

@Deprecated
public class SimpleDraweeView extends up6 {
    public static r2f z;
    public p0 y;

    public SimpleDraweeView(Context context) {
        super(context);
        l(context, (AttributeSet) null);
    }

    public p0 getControllerBuilder() {
        return this.y;
    }

    public final void l(Context context, AttributeSet attributeSet) {
        TypedArray obtainStyledAttributes;
        int resourceId;
        try {
            tf6.P();
            if (isInEditMode()) {
                getTopLevelDrawable().setVisible(true, false);
                getTopLevelDrawable().invalidateSelf();
            } else {
                cvg.p(z, "SimpleDraweeView was not initialized!");
                this.y = (p0) z.get();
            }
            if (attributeSet != null) {
                obtainStyledAttributes = context.obtainStyledAttributes(attributeSet, qnc.SimpleDraweeView);
                if (obtainStyledAttributes.hasValue(qnc.SimpleDraweeView_actualImageUri)) {
                    m(Uri.parse(obtainStyledAttributes.getString(qnc.SimpleDraweeView_actualImageUri)));
                } else if (obtainStyledAttributes.hasValue(qnc.SimpleDraweeView_actualImageResource) && (resourceId = obtainStyledAttributes.getResourceId(qnc.SimpleDraweeView_actualImageResource, -1)) != -1) {
                    if (isInEditMode()) {
                        setImageResource(resourceId);
                    } else {
                        setActualImageResource(resourceId);
                    }
                }
                obtainStyledAttributes.recycle();
            }
            tf6.P();
        } catch (Throwable th) {
            tf6.P();
            throw th;
        }
    }

    public final void m(Uri uri) {
        p0 p0Var = this.y;
        p0Var.d = null;
        knb knb = (knb) p0Var;
        knb.c(uri);
        knb.l = getController();
        setController(knb.a());
    }

    public void setActualImageResource(int i) {
        m(mzf.c(i));
    }

    public void setImageRequest(qa7 qa7) {
        p0 p0Var = this.y;
        p0Var.e = qa7;
        p0Var.l = getController();
        setController(p0Var.a());
    }

    public void setImageResource(int i) {
        super.setImageResource(i);
    }

    public void setImageURI(Uri uri) {
        m(uri);
    }

    public void setImageURI(String str) {
        m(str != null ? Uri.parse(str) : null);
    }

    public SimpleDraweeView(Context context, AttributeSet attributeSet) {
        super(context, attributeSet);
        l(context, attributeSet);
    }

    public SimpleDraweeView(Context context, AttributeSet attributeSet, int i) {
        super(context, attributeSet, i);
        l(context, attributeSet);
    }
}
